src/app/features/genes/components/gene-nominated-targets/gene-nominated-targets.component.spec.ts

```diff
@@ -29,10 +38,109 @@ describe('Component: Gene Nominated Targets', () => {
   beforeEach(async () => {
     fixture = TestBed.createComponent(GeneNominatedTargetsComponent);
     component = fixture.componentInstance;
-    fixture.detectChanges();
   });
 
+  const setUp = (genes: Gene[]) => {
+    const genesResponse: GenesResponse = {
+      items: genes
+    };
+    mockApiService = TestBed.inject(ApiService);
+    spyOn(mockApiService, 'getNominatedGenes').and.returnValue(
+      of(genesResponse)
+    );
+    fixture.detectChanges();
+    element = fixture.nativeElement;
+
+    expect(mockApiService.getNominatedGenes).toHaveBeenCalled();
+
+    const table = element.querySelector('table');
+    expect(table).not.toBeNull();
+
+    const rows = Array.from(
+      table?.querySelectorAll('tbody tr') || []
+    ) as HTMLTableRowElement[];
+
+    return { rows };
+  };
```

Comment:
Create `setUp` method so that each test can specify which genes should be returned by the mock API service